app/views/errors/internal_server.html.erb

```diff
@@ -0,0 +1,3 @@
+<%# app/views/errors/internal_server.html.erb %>
```

Comment:
Same thing regarding the licensing here. Please replace these lines in the top of the new erb files with:

```ruby
<%- # License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
# Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE -%>
```